<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureWidget</class>
 <widget class="QWidget" name="CaptureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>854</width>
     <height>370</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>626</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btnStop">
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:rgb(255, 0, 0);
color:red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>主机</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHost"/>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort"/>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>最小长度限制</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLength"/>
   <widget class="QPushButton" name="btnStart">
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAdvance">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>高级选项</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxInter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_inter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>网络接口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
